ies-route53/recipes/add.rb

```diff
@@ -20,7 +20,7 @@
   aws_secret_access_key node['ies-route53']['zone']['custom_secret_key']
   overwrite true
   action :create
-  not_if do
-    node.fetch('ies-route53', {}).fetch('zone', {}).fetch('id', {}).nil?
```

Comment:
This line was completely off.

Here is what it does:

 1. Look for `ies-route53`, if you can't find it, fetch an object
 2. Look for `zone` in `ies-route53`, if you can't find it, fetch an object
 3. Look for `id` in `zone`, if you can't find it, fetch an object

Now, add this to the default attributes we set:

```
default[...]['zone']['id'] = ''
```

TL;DR — `id` is never nil. It could be `.empty?`, or if someone somehow deletes all default attributes, it is an object! :)